Doc/ota 2578/logging (#561)
advancedtelematic/meta-updater: README.adoc

== Development configuration

=== Logging

During development, it is recommended to enable persistent systemd logging in case you run into problems. To do so, add this to your `local.conf`:

....

> IMAGE_INSTALL_append += " systemd-journald-persistent"

Reviewer: Note that this is enabled by default for newly configured builds: https://github.com/advancedtelematic/meta-updater/blob/7530d93ebb97b519a12b88d121320ac0bb711c46/conf/local.conf.sample.append#L26. Maybe the phrasing could mention it?

Author: Done. Also added a reference to historical support for the porter board.

== Development configuration

=== Logging

> During development, it is recommended to enable persistent systemd logging in case you run into problems. This is enabled by default for newly configured environments (see link:conf/local.conf.sample.append[]). To enable it manually, put this to your `local.conf`:

Reviewer 2: ```suggestion
 To troubleshoot problems that you might encounter during development, we recommend that you enable persistent `systemd` logging. This setting is enabled by default for newly configured environments (see link:conf/local.conf.sample.append[]). To enable it manually, put this to your `local.conf`:
```

Author: Good call. I've incorporated your suggestion.
